arkane/gaussian.py

```diff
@@ -282,48 +280,47 @@ def loadEnergy(self, frequencyScaleFactor=1.):
                 # hence to get the correct Elec from E (0 K) we need to subtract the scaled ZPE
 
                 elif 'E(ZPE)' in line:
-                    scaledZPE = float(line.split()[1]) * constants.E_h * constants.Na
+                    scaled_zpe = float(line.split()[1]) * constants.E_h * constants.Na
                 elif '\\ZeroPoint=' in line:
                     line = line.strip() + f.readline().strip()
                     start = line.find('\\ZeroPoint=') + 11
                     end = line.find('\\', start)
-                    scaledZPE = float(line[start:end]) * constants.E_h * constants.Na * frequencyScaleFactor
+                    scaled_zpe = float(line[start:end]) * constants.E_h * constants.Na * frequencyScaleFactor
```

Comment:
You're passing your newly defined `zpe_scale_factor` to this method in `statmech.py` but here we're removing the scaled ZPE from the composite E0. Are we sure that should be the ZPE scale factor and not the frequency scale factor?

Or maybe the more important question is, why are we actually doing this? As far as I can tell, `\ZeroPoint` gets printed in the archive string at the end of Gaussian jobs, but if it's a composite method, the last occurrence of ZPE will be from the `E(ZPE)` string instead. Therefore, I am wondering if the ZPE as defined on line 288 ever gets used. Do you know? If it does get used, then why are we multiplying it by the scale factor? Shouldn't it be the same as the statement on line 283?

Reply:
Thanks for the explanation. This clears up my confusion.